medical-system/src/auth/hooks/useColumns.tsx

```diff
@@ -0,0 +1,29 @@
+import { ColumnDef } from "@tanstack/react-table";
+import { User } from "../../components/shared/Table";
+
+export const useColumns = (): ColumnDef<User>[] => {
+  const columns: ColumnDef<User>[] = [
+    {
+      header: "Name",
+      accessorKey: "name",
+    },
+    {
+      header: "Email",
+      accessorKey: "email",
+    },
+    {
+      header: "Mobile",
+      accessorKey: "mobile",
+    },
+    {
+      header: "Address",
+      accessorKey: "address",
+    },
+    {
+      header: "Date Birth",
+      accessorKey: "dateBirth",
```

Comment:
Jak wyświetlasz lub pobierasz datę? W jaki sposób zapisujesz datę podczas tworzenia pacjenta? Zwykle zapisuje się datę w formacie ISO, a później aby ją wyświetlić w formacie dla usera to używamy bibliotek do pracy z datami np date-fns.

Poza tym, zastanawiam się czy to dobre miejsce na taki hook bo jest to folder auth.